docs/reference/setup/install/targz-security.asciidoc

```diff
@@ -0,0 +1,68 @@
+[role="exclude"]
+==== Start {es} with security enabled
```

Comment:
The title reads as if this is an option among many but it is actually the default behaviour , maybe we could choose another ? 

I also find it slightly confusing that this section comes _after_ we tell users how to start Elasticsearch. Another point is that we don’t tell users at all that they can start a node with —enrolment-token so that it joins an existing cluster. Maybe this is not the place (?) but can we at least link to a page that explains how to get a multi node cluster running with security auto configuration and the enrolment process ?

Reply:
Unfortunately, the first section for "running from the command line" is reused in this page for [Starting Elasticsearch](https://www.elastic.co/guide/en/elasticsearch/reference/master/starting-elasticsearch.html#_running_elasticsearch_from_the_command_line). So if we update that section, it gets updated in both places. Maybe that's what we want, now that starting Elasticsearch with security ON by default is the norm.

Reply:
I think it makes sense tbh. The earlier we tell users what to expect when they start es , the better . Maybe a short reference there so that it is pointed out everywhere and then we can elaborate on the security section ?